task/eaas-provision-space/0.1/eaas-provision-space.yaml

```
@@ -5,26 +5,26 @@ metadata:
   name: eaas-provision-space
 spec:
   description: >-
-    Provisions an ephemeral namespace on an EaaS cluster using a SpaceRequest.
+    Provisions an ephemeral namespace on an EaaS cluster using a crossplane namespace claims.
```

```suggestion
    Provisions an ephemeral namespace on an EaaS cluster using a crossplane namespace claim.
```

Same applies to the other descriptions.